normal.sql

```diff
@@ -0,0 +1,147 @@
+-- Get the average rating for a movie
+SELECT title, avg(rating)
```

Comment:
As mentioned in class aliases will help clean up the return column name. You can even reference the alias name in the `ORDER BY` clause